### ISSUE TITLE
crash when opening same .v in feedback window

### ISSUE THREAD

**Issue by andres-erbsen-sifive** [whonore/Coqtail#44]
open a file, CoqToCursor to the middle of it, then open the same file in the bottom-right window:

```
Error detected while processing function coqtail#OpenPanels:
line   19:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/andres-erbsen/.vim/pack/coq/start/coqtail/python/coqtail.py", line 590, in reset_color
    vim.command("call coqtail#ClearHighlight()")
```

this happens even if i split the bottom-right window first

**Comment by whonore** [comment]
Yea the way things are currently it expects there to be just the main, info, and goal buffer and things break pretty fast if that's not the case. I'll look into relaxing those assumptions soon.

**Comment by whonore** [comment]
Fixed in [1.0.0](https://github.com/whonore/Coqtail/tree/v1.0.0)
